client/src/containers/loginPage/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import { Form, Input, Button } from 'antd';
+import { UserOutlined, LockOutlined } from '@ant-design/icons';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import * as ROUTES from '../../constants/routes';
+
+import logo from '../../assets/images/login-logo.png';
+import loginImg from '../../assets/images/Group 369.svg';
+
+import './style.css';
+
+const LoginPage = ({ updateAuth, history }) => {
+  const onFinish = async (values) => {
+    const { push } = history;
+    try {
+      const {
+        data: { statusCode },
+      } = await axios.post('/api/v1/login', values);
+      if (statusCode === 200) {
+        updateAuth();
+        push(ROUTES.HOME_PAGE);
+      }
+    } catch (err) {
+      console.log('error', err);
```

Comment:
Here it is better to add notification indicating the error condition instead of console.log
```suggestion
```